### PR TITLE
KT Active Martial Feats added + Artificer (Revised) updates

### PR DESCRIPTION
**Pull request by patwells**
KT Active Martial Feats

### CODE REVIEW

**Comment by patwells** [comment]
Made updates to KT Arti (Revised). Barebones but making updates.

File: class/KibblesTasty; Artificer (Revised).json [hunk 1]
```diff
@@ -1,7569 +1,7831 @@
...
-							"Warplate",
-							"18",
-							"75 lbs.",
-							"-",
-							"Disadvantage",
-							"Heavy Armor, Powerful Build*"
-						],
-						[
-							"Warsuit",
-
```
Comment (TheGiddyLimit) [inline class/KibblesTasty; Artificer (Revised).json:7528]:
```suggestion
```

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]

File: class/KibblesTasty; Artificer (Revised).json (hunk 1 above)
Comment (TheGiddyLimit) [reply class/KibblesTasty; Artificer (Revised).json:7528]: 🤔 either I've messed something up, or GitHub is failing to deal with this massive file. 
Either way, you've got a nameless, text-less feature in there that you need to remove.

Comment (revilowaldow) [reply class/KibblesTasty; Artificer (Revised).json:7528]: If you're trying to give suggestions on a huge file then github gets lines wrong yeah. :(

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
might I recommend running the tests locally

1) (clone the repo)
1) install [NodeJS](https://nodejs.org/en/)
1) open a command prompt in your folder
1) `npm i`
1) `npm t` (fix + repeat until it works)

**Comment by revilowaldow** [comment]
This should work, remember to change the extension to `.json`
[KibblesTasty; Artificer (Revised).txt](https://github.com/TheGiddyLimit/homebrew/files/8643851/KibblesTasty.Artificer.Revised.txt)

**Pull request merged**